lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -65,14 +65,31 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
     active_closures = Enum.flat_map(active, &elevator_closure/1)
     at_this_elevator? = fn %Closure{id: id} -> id == elevator_id end
 
+    {:ok, %Stop{id: stop_id}} = @facility.fetch_stop_for_facility(elevator_id)
+
+    relevant_closures =
+      active_closures |> Enum.filter(&relevant_closure?(&1, stop_id, active_closures))
```

Comment:
Conventional Elixir tends not to use a pipe operator when only a single function is being piped-into:

```elixir
Enum.filter(active_closures, &relevant_closure?(&1, stop_id, active_closures))
```